content/ConnectingToDatabase/ConnectingToDatabase.facts.md

```diff
@@ -0,0 +1,56 @@
+### Facts for Connecting to Database lesson:
+
+`PDO_PGSQL` driver is a tool of `PHP Data Objects (PDO) Interface` to allow connection between PHP and PostgreSQL database.
+
+`PDO` class represents a connection among PHP and a database server.
+
+In the example code below shows how PHP uses the `PDO_PGSQL` driver and `PDO` class to connect to the PostgreSQL database.
+
+Code:
+
+```php
+<?php
+    $host='localhost';
+    $port = '5432';
+    $dbname = 'MyDatabase';
+    $username = 'postgres';
+    $password = 'Admin01';
+    
+    $dsn = "pgsql:host=$host;port=$port;dbname=$db;user=$username;password=$password";
+
+    try{
```

Comment:
Please follow the PSR 2 coding guidelines for proper way of writing PHP codes. Please check this link. Thanks.

https://www.php-fig.org/psr/psr-2/